webdriver-factory/pom.xml

```diff
@@ -54,7 +111,18 @@
 
         <dependency>
             <groupId>org.seleniumhq.selenium</groupId>
+            <artifactId>selenium-api</artifactId>
```

Comment:
Is this needed? I thought that the selenium-api is a transitive dependency of selenium-java.